### ISSUE TITLE
Some listviews are not accessible with screen readers

### ISSUE THREAD

**Issue by adkurz** [notepad-plus-plus/notepad-plus-plus#2111]
In some dialogs, for example the pluginmanager, notepad++ uses a tool named Babygrit. This tool isn't accessible with screen readers like NVDA or JAWS.

**Comment by Robert-J-H** [comment]
Yes, this behaviour is very annoying.

I've just worked some hours on a NVDA add-on for Notepad++ that should enable the usage of the shortcut mapper.
It is nigh impossible to bring some functionality to this odd 'BABYGRID'
Screen reader users hear only 'Pane' when they move into the shortcut list.
The child of this pane is a window and its child is a list. However, the shortcuts are the items in this list.
Unfortunately, the name of such an item is not useable per se either because it is just all items in succession, prepended by the grid coordinates.
So it is something like:
u'000-001 | Name'
(recently, there has been added a category this would be a '003' as the second number).
I can essentially redirect the focus to those items and the user can navigate around with up/down arrow.
But note that it goes actually from left to right and then top to bottom and not down by rows.
Thus, you'll hear:
001-001 New
001-002 'Ctrl N'
'001-003 'File'

What's worse, it is reported that this item is selected and if you move to 'modify' the shortcut won't appear but rather the first one instead.
Dragging the mouse to this item and clicking does not help a lot because the item location is wrongly reported.

Anyway, I would be glad if this could be addressed, particularly since NVDA recommends Notepad++ for e.g add-on development.
Best
RJ

**Comment by MetaChuh** [comment]
please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.
if your issue still persists, and it is still of relevance to you, please feel free to file a new issue as easy to understand and reproducible as possible.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by Robert-J-H** [comment]
Well, I can't say that the issue is solved.
I've just checked if the shortcut list is now accessible but it
doesn't seem to be.
There's only a "Pane" reported.
Moving around with arrow keys does nothing (well, it would kinda but
without feedback, it's for naught).
In other words, one can read the list with our add-on enabled but
nothing more, not even selecting an entry in order to modify it.
I guess it is the same with the other dialogs where the baby grid class is used.
In summary, closing the issue doesn't change anything.

**Comment by sasumner** [comment]
>closing the issue doesn't change anything

Well, it gets it off somebody's plate. :)

Notepad++ is someone's unpaid hobby project, so some corner features are not possible to address.  Accessibility is one of these.  

Before reading this issue, I had never heard about "babygrid", but I have noticed odd behavior concerning the Shortcut Mapper:  Because there is no great way to export a list of all of the keycombos, in the past I have wanted to take screenshots for later use.  Unfortunately, with my screenshot program (SnagIt), when I invoke it, the Shortcut Mapper window autoscrolls to the top.  So it is only possible for me to get a screenshot of the first page of shortcuts.  I'm presuming this is because of some interaction issue with the "babygrid", so for this reason I would be in favor of something better being used here.

**Comment by Robert-J-H** [comment]
I would suggest to take a look on how it is done in Reaper.
The current shortcut assignments can even be viewed as a web page,
thus no need for taking a snapshot.
Best
Robert
